docs/worker-node/install-singularity.md

```diff
@@ -49,7 +53,10 @@ As with all OSG software installations, there are some one-time (per host) steps
 
 If the operating system is an EL 7 variant, and has been updated to EL
 7.4 or the 7.4 kernel (3.10.0-693 or greater), you can skip
-installation altogether and use singularity through CVMFS:
+installation altogether and instead do these steps to enable
+singularity to be run as an unprivileged user via CVMFS (although note
+that VO pilot jobs that use singularity may not yet be ready to use it
+that way so you may want both this and an rpm for a transition period):
```

Comment:
Instead of the parenthetical, add a sentence to the effect of:

> Singularity via CVMFS is a new method of distributing and not all VOs currently support it. To provide wider singularity support, consider installing the singularity RPM (link to section) as well as offering it via CVMFS.